### ISSUE TITLE
Vampires (is this channel even necessary?)

### ISSUE THREAD

**Issue by ThomasRubycrust** [McTsts/Werewolves-Roles#653]

**Comment by ThomasRubycrust** [comment]
Yeah, but none of the other teams have a dedicated channel for a specific type of group member.

**Comment by venomousbirds** [comment]
maybe bat should be added to the vamp channel then

**Comment by ThomasRubycrust** [comment]
Bat isn't a vampire though... If you want a channel like that I suggest we name it something different (but unsure what)

**Comment by ThomasRubycrust** [comment]
(the same is then also needed for Graveyard)
